src/Emitter.php

```diff
@@ -200,12 +201,13 @@ public function getRequestType($type) {
      * not exists already.
      *
      * @param string $dir - The directory we want to make
+     * @param bool   $isRecursive
      * @return bool|string - Boolean describing if the creation was a success
      */
-    public function makeDir($dir) {
+    public function makeDir($dir, $isRecursive = false) {
         try {
             if (!is_dir($dir)) {
-                mkdir($dir);
+                mkdir($dir, 0777, $isRecursive);
```

Comment:
maybe change it to `return mkdir($dir, 0777, $isRecursive);` if you touch this part